docs/eventing/samples/iot-core/README.md

```diff
@@ -199,37 +173,39 @@ see them in the subscriber.
         -events 10
     ```
 
+1. Inspect the logs of the subscriber:
+
+    ```shell
+    kubectl logs --selector serving.knative.dev/service=message-dumper -c user-container
+    ```
+
+    You should see something along the similar to:
+    
+    ```shell
+    {"ID":"481014114648052","Data":"eyJzb3VyY2VfaWQiOiJpb3QtY29yZSBkZW1vIiwiZXZlbnRfaWQiOiJlaWQtMzI3MjJiMzItZWU5Mi00YzZlLWEzOTgtNDlmYjRkYWYyNGE1IiwiZXZlbnRfdHMiOjE1NTM3MTczOTYsIm1ldHJpYyI6MC4xMzY1MjI5OH0=","Attributes":{"deviceId":"iot-demo-client","deviceNumId":"2754785852315736","deviceRegistryId":"iot-demo","deviceRegistryLocation":"us-central1","projectId":"s9-demo","subFolder":""},"PublishTime":"2019-03-27T20:09:56.685Z"}
+
 ### Cleanup
 
 To cleanup the knative resources:
 
-1.  Remove the gcppubsub controller:
+1.  Remove the `GcpPubSubSource`:
```

Comment:
Do users need to replace anything in this command before running it? If so, it's good to call out what needs to be replaced with the user's data.

Reply:
No, the command is self-contained. It requires no changes to the YAML ahead of time. 

It does require that the user runs the command in a shell with the proper environment variables defined. See the "Environment Variables" section.